tools/cli/go-whisk-cli/commands/activation.go

```diff
@@ -217,8 +217,8 @@ var activationResultCmd = &cobra.Command{
 }
 
 var activationPollCmd = &cobra.Command{
-    Use:   "poll [NAMESPACE]",
-    Short: wski18n.T("poll continuously for log messages from currently running actions"),
+    Use:   "poll [ NAMESPACE | ENTITY_NAME ]",
```

Comment:
since the list is limited, i think the more explicit `poll [NAMESPACE | ACTION_NAME | TRIGGER_NAME | RULE_NAME]` is helpful